dbmgr/dbmgr.go

```diff
@@ -328,7 +329,7 @@ func NewAuthTokensQuery(db *gorm.DB) *AuthTokensQuery {
 }
 
 func (q *AuthTokensQuery) Create(authToken AuthToken) error {
-	return q.DB.Create(&authToken).Error
+	return q.DB.Omit(constants.TokenColumnName).Create(&authToken).Error
```

Comment:
instead of using omit, which requires too many changes, why not ignore this field directly in the mode using `gorm: "-"` ?

Reply:
We still want to respect existing tokens, and simply warn users in the FE if their token is stored raw and should be replaced for increased security. `gorm: "-"` would prevent reading any existing tokens and make them effectively inaccessible. However, because you asked i discovered `gorm:"->"` which marks a field as read only and achieves what we want without using omit. Updated the PR to use the `->` tag.

Reply:
nice, I didn't know about `->` 😸